jest-reporters/emit-only-failures.js

```
@@ -0,0 +1,21 @@
+/* eslint-disable no-param-reassign */
+// eslint-disable-next-line import/no-extraneous-dependencies
```

ESLint

[error] Definition for rule 'import/no-extraneous-dependencies' was not found.

(import/no-extraneous-dependencies)